minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/MalmoModClient.java

```diff
@@ -251,8 +252,10 @@ public void onRightClickEvent(PlayerInteractEvent.RightClickBlock event){
     public void onGuiOpenEvent(GuiOpenEvent event){
         if(this.stateMachine.getStableState() == ClientState.RUNNING){
             Logger logger = Logger.getLogger("MalmoModClient.onGuiOpenEvent");
-            if (event != null && event.getGui() != null)
+            if (event != null && event.getGui() != null && !(event.getGui() instanceof GuiGameOver)) {
                 logger.log(Level.WARNING, "GUI Window " + event.getGui().getClass().getSimpleName() + " opened!");
+                throw new AssertionError("GUI Window " + event.getGui().getClass().getSimpleName() + " opened!");
```

Comment:
If we should not have gui windows open, lets be extreme about it

Reply:
Fine for now. That said, I could imagine a future in which we actually want to train an agent to use the gui interfaces. But not yet.